app/src/main/java/org/fossasia/openevent/general/similarEvents/EventTopicsDao.kt

```diff
@@ -0,0 +1,20 @@
+package org.fossasia.openevent.general.similarEvents
+
+import android.arch.persistence.room.Dao
+import android.arch.persistence.room.Insert
+import android.arch.persistence.room.OnConflictStrategy
+import android.arch.persistence.room.Query
+import io.reactivex.Flowable
+import org.fossasia.openevent.general.event.Event
+
+@Dao
+interface EventTopicsDao {
+    @Insert(onConflict = OnConflictStrategy.REPLACE)
+    fun insertSimilarEvents(events: List<Event>)
+
+    @Query("DELETE FROM Event")
+    fun deleteAllEvents()
+
+    @Query("SELECT * from Event WHERE eventTopic = :topicId")
+    fun getAllSimilarEvents(topicId: Long): Flowable<List<Event>>
```

Comment:
Add this in EventDao, you are selecting from the same table, so should remain in the same DAO

By this logic, we should have FavoriteDao as well